### ISSUE TITLE
Test: Check if game service is working with player's and computer's options

### ISSUE THREAD

**Issue by StraykerPL** [StraykerPL/RockPaperScissors#39]
**Is your feature request related to a problem? Please describe.**
After both of sides made their choices (player and computer) website must determine who won.

**Describe the solution you'd like**

- If player's choice is winning, show victory message to user and increment score counter,
- If computer's choice is winning, show lose message to user and do not increment score counter,
- If none of the choices is winning (draw), give victory to player (draw is not supported, out of scope for this project),

**Describe alternatives you've considered**

**Additional context**

**Comment by StraykerPL** [comment]
This issue needs lable from #42

**Comment by StraykerPL** [comment]
Blocked by #37 #38

**Comment by StraykerPL** [comment]
Label from #42 implemented

**Comment by StraykerPL** [comment]
Unblocked

**Comment by StraykerPL** [comment]
Tested, everything's fine
